lookout/style/format/tests/test_model.py

```diff
@@ -7,6 +7,26 @@
 from lookout.style.format.tests.test_rules import load_abalone_data
 
 
+def compare_models(test_case: unittest.TestCase,
+                   format_model1: FormatModel,
+                   format_model2: FormatModel):
+    test_case.assertEqual(format_model1.languages, format_model2.languages)
+    for lang in format_model1.languages:
```

Comment:
should we add a comparison operator to `FormatModel`?

Reply:
it is hard to tell because you can want different things. If you want to find out that models are from the same source, you should check metadata. if you want to check if they have the same rules, you should check rules only. And for parameters equality, you check them also. So, IMO, it is better to keep it without comparison operator until we have the real case for it.